stories/RangeSlider/examples/TipFormatter.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import RangeSlider from '../../../components/RangeSlider';
+
+const TipFormatter = aditionalProps => (
+  <RangeSlider tipFormatter={value => `R$ ${value}`} {...aditionalProps} />
+);
+
+TipFormatter.code = '<RangeSlider tipFormatter={value => `R$ ${value}`} />'; // eslint-disable-line
```

Comment:
Why is that `disable line`?

Reply:
Because this is used to create an example of the component's code. And eslint acuses an error when you put `${value}` inside a simple `'`.